locales/ja_JP/LC_MESSAGES/duckduckgo.po

```diff
@@ -4470,7 +4470,7 @@ msgstr "リンク"
 
 msgctxt "maps_places"
 msgid "Website"
-msgstr ""
+msgstr "ウェブサイト"
```

Comment:
I think that it is certainly good as one idea to translate it as a ホームページ.

However, I think that it is better to translate it as directly as possible so that the meaning does not change.

(ja: 私は、「ホームページ」という単語はトップページを意味すると思います。この単語は地図に付属する情報として表示されていますが、ここに入るURLが常にホームページ(トップページ)のものであるか分からないため、ウェブページと訳した方が無難であると思います。)

Reply:
Ah, I see what you mean. Thanks for the explanation. In that case if you think ウェブページ would be better, that sounds like a good choice to me.